src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/IncidentCorrectionRequest.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.incidentreporting.jpa
+
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.ManyToOne
+import java.time.LocalDateTime
+
+@Entity
+class IncidentCorrectionRequest(
```

Comment:
do you think this will need to have a historical vs. modern (/new/whatever we call the future) version?